pontoon/checks/libraries/pontoon.py

```diff
@@ -33,7 +35,7 @@ def run_checks(entity, string):
     Group all checks related to the base UI
 
     :arg pontoon.base.models.Entity entity: Source entity
-    :arg basestring string: a translation
+    :arg six.string_types string: a translation
```

Comment:
@adngdb Do you prefer the previous version?

Reply:
I think it would better to be less specific here. Hopefully once we're moved to py3 we should remove `six`, right?

Reply:
That's right, I'll revert it then.